metaflow/plugins/packager_decorator.py

```diff
@@ -0,0 +1,15 @@
+import os
+from pathlib import Path
+from metaflow.metaflow_config import DEFAULT_PACKAGE_SUFFIXES
```

Comment:
from pathlib import Path
from metaflow.metaflow_config import DEFAULT_PACKAGE_SUFFIXES 
was not used。